libp2p/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 - Add `libp2p-quic` stable release.
 
+- Update `libp2p-identity` to `0.2.3`.
+
```

Comment:
```suggestion
```

I'd say let's not bother with these entries if we don't have to. They are just noise.

Reply:
ok, updated :)